client/my-sites/plugins/plugin-eligibility/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import MainComponent from 'components/main';
+import HeaderCake from 'components/header-cake';
+import EligibilityWarnings from '../eligibility-warnings';
+
+class PluginEligibility extends Component {
+	static propTypes = {
+		pluginSlug: PropTypes.string,
+		siteSlug: PropTypes.string,
+		translate: PropTypes.func,
+	};
+
+	getBackUrl = () => {
+		const { pluginSlug, siteSlug } = this.props;
+
+		return `/plugins/${ pluginSlug }/${ siteSlug }`;
+	};
+
+	goBack = () => page( this.getBackUrl() );
+
+	render() {
+		const { translate } = this.props;
+
+		// Mock some errors and warnings until we get real data from the API
```

Comment:
once we get #9914 merged, we can create a snippet in Chrome's dev tools with this state and directly run that from the console. that way we can easily fake the state without even polling or mocking it

```js
dispatch( {
	type: actionTypes.AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE,
	status: 'conflicts',
	lastUpdate: Date.now(),
	eligibilityHolds: [ { … }, { … }, { … } ],
	eligibilityWarnings: [ … ]
} );
```